### ISSUE TITLE
Unable to open 'Preview 'manual.rst'': Error

### ISSUE THREAD

**Issue by rksys** [vscode-restructuredtext/vscode-restructuredtext#64]
Please follow [the troubleshooting guide](https://github.com/vscode-restructuredtext/vscode-restructuredtext/blob/master/docs/sphinx.md#troubleshooting-guide) to locate the culprit.

If troubleshooting does not reveal any useful information, provide the following information to create a new issue,

**Environment**
- [x] Extension version: 32.0.0
- [x] Visual Studio Code version: Version 1.17.1 (1.17.1)
- [x] Operating System version: OSX EI Capitan
- [x] Python version:3.6
- [x] sphinx version:1.6.4
- [x] A sample of conf.py or its important fragments:
    *  `  "restructuredtext.confPath": "${workspaceRoot}",`
    * I am a newbie to rst, thus only ran sphinx-quickstart with default,  and don't have any `config.py` yet.  
    * the `ls -a` output is : `Makefile   build      make.bat   manual.rst source
`

**Expected behavior**:

the preview should works

**Actual behavior**:

the preview window show up 1 ms and then flash out, with vscode error info: 

```
Unable to open 'Preview 'manual.rst'': Error
```

**Detailed Steps to reproduce the problem**:

basically just by open [manual.rst](https://pandoc.org/demo/MANUAL.txt) and press `Cmd-k r`

**Comment by lextm** [comment]
https://github.com/vscode-restructuredtext/vscode-restructuredtext/blob/master/docs/sphinx.md#live-preview-settings

You have to use
```
{
    "restructuredtext.builtDocumentationPath" : "${workspaceRoot}/build/html",
    "restructuredtext.confPath"               : "${workspaceRoot}/source"
}
```
and `manual.rst` must be put into `source` folder, where your `config.py` should also be.
